recipes/graphviz/run_test.py

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env python
```

Comment:
I'll do it after we have new packages for Graphviz and it's Python bindings. Then I plan to move this test to the Python package.

In any case, it was useful for me to have it here because I noticed that it was failing on Windows (even when the Graphviz tests alone were working correctly :-).